src/styles/blocks/card.scss

```diff
@@ -0,0 +1,118 @@
+.card {
+  width: $widthCard;
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+
+  background: $background-color;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &--margin-right-bottom {
+    margin-right: $marginRightCard;
+    margin-bottom: 46px;
+  }
```

Comment:
that is called [atomic css](https://css-tricks.com/lets-define-exactly-atomic-css/)
I would say you should not add classes like that, because from them I don't understand what exactly piece of code is that

Reply:
In creating this class I try to avoid changing the main class "card" by margins. I've tried to save natural borders in the card.

I'll have deleted it)